notebooks/cleaning_data.ipynb

```diff
@@ -0,0 +1,2892 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    GLAD.plot_indirect_missing(indirect_missing)
</pre><p>Do comment on what this does</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mawada-sweis/Clustering-Analysis/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>